dotcom-rendering/src/components/Masthead/Titlepiece/Logo.tsx

```diff
@@ -1,22 +1,152 @@
-/**
- * @file
- * This file was largely copied from src/components/Logo.tsx
- */
 import { css } from '@emotion/react';
-import { visuallyHidden } from '@guardian/source/foundations';
-import { SvgGuardianLogo } from '@guardian/source/react-components';
+import { from, space, visuallyHidden } from '@guardian/source/foundations';
+import { Hide, SvgGuardianLogo } from '@guardian/source/react-components';
 import { nestedOphanComponents } from '../../../lib/ophan-helpers';
 import { palette } from '../../../palette';
+import TheWholePictureGuardianLogoSmall from '../../../static/icons/the-guardian-whole-picture-logo-small.svg';
+import TheWholePictureGuardianLogo from '../../../static/icons/the-guardian-whole-picture-logo.svg';
 
-export const Logo = () => (
-	<a href="/" data-link-name={nestedOphanComponents('header', 'logo')}>
-		<span
-			css={css`
-				${visuallyHidden};
-			`}
-		>
-			The Guardian - Back to home
-		</span>
-		<SvgGuardianLogo textColor={palette('--masthead-nav-link-text')} />
-	</a>
+const gridMainColumn = css`
```

Comment:
From a refactor of the Logo component. The containing div is now part of this component. This refactor is contained within the first commit of this PR: https://github.com/guardian/dotcom-rendering/pull/14527/commits/070c22fc3697492445936b8fbdeae3e4d348f808